drizzlepac/haputils/catalog_utils.py

```diff
@@ -1479,10 +1477,12 @@ def identify_sources(self, **pars):
 
                 # Compute the ratio of big sources/islands using Custom/Gaussian kernel vs Rickerwavelet kernel
                 # This value can be used as a discriminant between overlapping point sources and nebulousity fields
-                ratio_cg2rw_bigsource = g_bs / rw_bs
+                ratio_cg2rw = 0.0
```

Comment:
Should this be 'ratio_cg2rw_bigsource' instead?

Reply:
Fixed in ada6517.